tasks/deploy/uninstall-task.yaml

```diff
@@ -8,6 +8,21 @@ spec:
       name: kubeconfig-path
       type: string
   steps:
+    - name: remove-leftovers-kuadrant-ns
+      args:
+        - >-
+          for crd in $(kubectl get crd -o name | grep "kuadrant" | sed 's/.*\/\(.*\)/\1/'); do
+              kubectl get --chunk-size=0 -o name -n "kuadrant" "$crd" |\
+              xargs --no-run-if-empty -P 20 -n 1 kubectl delete --ignore-not-found -n "kuadrant"
```

Comment:
Would not it be better to have the namespace name parametrized and keeping "kuadrant" as default only. Some pipelines (e.g. testsuite-pipeline) have 'project' as parameter where you can specify the namespace name and "kuadrant" is just default.

Reply:
The truth is that that pipeline does not use this task so maybe I am overcomplicating things?

Reply:
Probably ok to keep it hard coded for now.